File: 程式語言 V2答案檔/C++/304.cpp
#include <iostream>
using namespace std;

int compute(int []);
int main()
{
    int i, input[6];
    for (i = 0; i < 6; i++)
        cin >> input[i];
    cout << compute(input);
    return (0);
}

int compute(int s[])
{
    int i, count = 0;
    for (i = 0; i < 6; i++) {
        if (s[i] % 3 == 0) {
            count++;
        }
    }
    return count;
}

File: 程式語言 V2答案檔/Go/304.go
package main
import "fmt"

func compute(array [6]int) int {
	count := 0
	for i := 0; i < 6; i++{
		if array[i] % 3 == 0{
			count += 1
		}
	}
	return count
}

func main(){
	var array [6]int
	for i := 0; i < 6; i++{
		fmt.Scanln(&array[i])
	}
	fmt.Println(compute(array))
}
